ci-operator/step-registry/openshift/e2e/cert-rotation/conf/assisted-on-gcp/openshift-e2e-cert-rotation-conf-assisted-on-gcp-commands.sh

```diff
@@ -66,11 +66,19 @@ if ! whoami &> /dev/null; then
 fi
 EOF
 
-# Prepare login via root
-LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
-    --zone "${GOOGLE_COMPUTE_ZONE}" \
-    packer@"${INSTANCE_PREFIX}" \
-    --command "sudo cp ~/.ssh/authorized_keys /root/.ssh && sudo sed 's;PermitRootLogin no;PermitRootLogin yes;g' -i /etc/ssh/sshd_config && sudo systemctl restart sshd"
+set +e
+for imagestream in $(seq 1 5)
```

Comment:
`seq` is unnecessary here， make it simple.
```suggestion
for imagestream in {1..5}
```